metacat-connector-druid/src/main/java/com/netflix/metacat/connector/druid/client/DruidHttpClientUtil.java

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright 2017 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ */
+
+package com.netflix.metacat.connector.druid.client;
+
+/**
+ * DruidHttpClientUtil.
+ *
+ * @author zhenl
+ * @since 1.1.0
+ */
+public final class DruidHttpClientUtil {
+    private DruidHttpClientUtil() {
+    }
+
+    /**
+     * get Latest Segment.
+     *
+     * @param input segments strings
+     * @return lastest segment id
+     */
+    public static String getLatestSegment(final String input) {
+        final String[] segments = input.substring(1, input.length() - 1).split(",");
```

Comment:
You might want to look at the Apache `StringUtils` class for some of these to save yourself the logic problems